hnn_core/drives.py

```diff
@@ -299,11 +299,10 @@ def _drive_cell_event_times(drive_type, dynamics, tstop, target_type='any',
     # brute force remove non-zero times. Might result in fewer vals
     # than desired
     # values MUST be sorted for VecStim()!
-    if len(event_times) > 0:
-        event_times = event_times[np.logical_and(event_times > 0,
-                                                 event_times <= tstop)]
-        event_times.sort()
-        event_times = event_times.tolist()
+    event_times = event_times[np.logical_and(event_times > 0,
+                                             event_times <= tstop)]
+    event_times.sort()
+    event_times = event_times.tolist()
```

Comment:
This isn't totally necessary to address the original issue, but I originally included it here to ensure that all spike trains (even empty ones) were converted to lists. We can revert if we feel it's out-of-place here.